server/src/main/java/org/opensearch/cluster/decommission/DecommissionAttributeMetadata.java

```diff
@@ -142,7 +142,7 @@ public String getWriteableName() {
 
     @Override
     public Version getMinimalSupportedVersion() {
-        return Version.V_3_0_0;
+        return Version.V_2_4_0;
```

Comment:
Is this the only place we have versioning enabled? What happens to clusters running mixed version 2.0 and 2.4 lets say

Reply:
Two more places - 
```
DECOMMISSIONING_FAILED_EXCEPTION(
            org.opensearch.cluster.decommission.DecommissioningFailedException.class,
            org.opensearch.cluster.decommission.DecommissioningFailedException::new,
            163,
            V_2_4_0
        ),
        NODE_DECOMMISSIONED_EXCEPTION(
            org.opensearch.cluster.decommission.NodeDecommissionedException.class,
            org.opensearch.cluster.decommission.NodeDecommissionedException::new,
            164,
            V_2_4_0
        );